### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by pyrech**
- Upgrade to Symfony 6.1
- Upgrade recipes

### CODE REVIEW

File: config/packages/monolog.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                level: debug
+                formatter: monolog.formatter.json
+            console:
+                type: console
+                process_psr_3_messages: false
+                channels: ["!event", "!doctrine"]
+            deprecation:
+                type: stream
+                channels: [deprecation]
+                path: php://stderr
```
Comment (damienalexandre) [inline config/packages/monolog.yaml:61]: We must use "error_log": https://www.clever-cloud.com/doc/deploy/application/php/tutorials/tutorial-symfony/#configure-monolog-to-get-application-logs

Comment (damienalexandre) [reply config/packages/monolog.yaml:61]: (it was wrong before)

File: config/routes.yaml [hunk 2]
```diff
@@ -12,8 +12,4 @@ sitemap:
 
 controllers:
     resource: ../src/Controller/
-    type: annotation
-
-kernel:
-    resource: ../src/Kernel.php
-    type: annotation
+    type: attribute
```
Comment (damienalexandre) [inline config/routes.yaml:15]: Did you change the controllers too? :thinking:

**Review by damienalexandre — CHANGES_REQUESTED** [review]

File: config/routes.yaml (hunk 2 above)
Comment (pyrech) [reply config/routes.yaml:15]: We already use attributes :)

File: config/packages/monolog.yaml (hunk 1 above)
Comment (pyrech) [reply config/packages/monolog.yaml]: Done

**Comment by damienalexandre** [comment]
Thanks a lot !

**Pull request merged**